gradle/kotlin-js-lib.gradle

```diff
@@ -2,22 +2,31 @@
  * Configure a Kotlin-JS project with safe defaults.
  */
 
-apply plugin: "org.jetbrains.kotlin.js"
+apply plugin: "org.jetbrains.kotlin.multiplatform"
```

Comment:
The kotlin.js plugin is being deprecated in favor of the kotlin multiplatform plugin.